packages/diagnostic/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
confirm this aligns with the latest PR's which "optimize" various configs and consume from global

Reply:
This PR need to be adjusted after those PRs merged